docs/tutorial/production-environment.md

```diff
@@ -51,12 +54,13 @@ admin     admin.prod.ancora.f.nchr.io
 adminssh  adminssh.prod.ancora.f.nchr.io
 www       www.prod.ancora.f.nchr.io
 --------  ------------------------------
-Connection to aux.ancora.f.nchr.io closed.
 ```
-Then simply activate the previous release:
+
+*Note* the autoscaling metrics in the prior description.  Fleet autoscales the
+application tier by default.
```

Comment:
"Application tier" is terminology we haven't used anywhere else. If we're going to use it, we should standardise on that in the rest of our docs.